cdap-data-fabric/src/main/java/io/cdap/cdap/store/OwnerTable.java

```diff
@@ -247,7 +247,7 @@ private String createRowKey(NamespacedEntityId namespacedEntityId) {
         builder.append(ROW_KEY_SEPARATOR);
         builder.append(ARTIFACT_VERSION_ROW_KEY_PREFIX);
         builder.append(ROW_KEY_SEPARATOR);
-        builder.append(scheduleId.getVersion());
+        builder.append(ApplicationId.DEFAULT_VERSION);
```

Comment:
this will be confusing for somebody looking at it later, would be good to add a comment on why we always put the default version here

Reply:
added a comment